### DISCUSSION TITLE
🧩 FEATURE REQUEST: labels for squad posts

### DISCUSSION THREAD

**Discussion by Kailash-Codes** [dailydotdev/daily#1092] [Feature Requests]
### Concise Description

While sharing post to squad, we may want to attach label like important ,critical, optimization, testing etc.

### Description

While we want to share post to our squad, we should also be able to attach label like important , critical , testing , optimization etc.
So that the reader can easily understand the context of the post, rather than visiting the site and knowing about that

### Solution Proposed

Add a button to select label, like the label while we are creating an issue in github.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @Kailash-Codes,

Thanks for the suggestion!
